pacu/modules/mq__enum/main.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+import argparse
+import time
+import json
+
+from pacu.core.lib import downloads_dir
+from pacu.core.lib import strip_lines
+from pacu import Main
+from copy import deepcopy
+
+module_info = {
+    "name": "mq__enum",
+    "author": "6a6f656c & h00die of nDepth Security",
+    "category": "ENUM",
+    "one_liner": "Listo and describe brokers",
+    "description": strip_lines(
+        """
+        This module will attempt to list and gather information from available brokers.
+        """
+    ),
+    "services": ["MQ"],
+    "prerequisite_modules": [],
+    "arguments_to_autocomplete": [],
+}
+
+parser = argparse.ArgumentParser(add_help=False, description=module_info["description"])
+
+parser.add_argument(
+    "--regions",
+    required=False,
+    default=None,
+    help=strip_lines(
+        """
+    One or more (comma separated) AWS regions in the format "us-east-1". Defaults to all session regions.
+"""
+    ),
+)
+
+
+def main(args, pacu_main: "Main"):
+    session = pacu_main.get_active_session()
+
+    # Don't modify these. They can be removed if you are not using the function.
+    args = parser.parse_args(args)
+    print = pacu_main.print
+
+    key_info = pacu_main.key_info
+    fetch_data = pacu_main.fetch_data
+
+    # End don't modify
+    get_regions = pacu_main.get_regions
+    if not args.regions:
+        regions = get_regions("mq")
+    else:
+        regions = args.regions.split(",")
+
+    summary_data = {}
+    summary_data["mq"] = {}
+
+    for region in regions:
+        print("Starting region {}...".format(region))
+        summary_data["mq"][region] = {}
+
+        try:
+            client = pacu_main.get_boto3_client("mq", region)
+        except Exception as error:
+            print("Unable to connect to MQ service. Error: {}".format(error))
+            continue
+
+        # Prepare output file to store MQ data
+        now = time.time()
+        outfile_path = str(downloads_dir() / f"mq_enum_{now}.json")
+
+        try:
+            response = client.list_brokers(
+                MaxResults=100,
+            )
+
+        except Exception as error:
+            print(
+                "Unable to list brokers; Check credentials or No brokers are available. Error: {}".format(
+                    error
+                )
+            )
+            continue
+        print("  Found {} brokers".format(len(response["BrokerSummaries"])))
+        for broker in response["BrokerSummaries"]:
```

Comment:
If using the paginator
```suggestion
        for response in page_iterator:
                for broker in response["BrokerSummaries"]:
```